components/menu/menu.js

```diff
@@ -256,7 +256,7 @@ class Menu extends HierarchicalViewMixin(LitElement) {
 		e.stopPropagation();
 
 		const startsWith = function(item, value) {
-			if (item.text && item.text.length > 0 && item.text.substr(0, 1) === value) {
+			if (item.text && item.text.length > 0 && item.text.toLowerCase().substr(0, 1) === value) {
```

Comment:
Nit: could lower case just one char instead of all the text.
```
item.text.substr(0, 1).toLowerCase()
```
That being said, this code only runs each time they hit a key... probably can't even measure the difference in such case.